io/zenoh-transport/src/unicast/establishment/open.rs

```diff
@@ -571,11 +570,7 @@ pub(crate) async fn open_link(
 
     let mut state = State {
         transport: StateTransport {
-            batch_size: manager
-                .config
-                .batch_size
-                .min(batch_size::UNICAST)
-                .min(link.config.batch.mtu),
+            batch_size: manager.config.batch_size.min(link.config.batch.mtu),
```

Comment:
I'd rather silent this warning. It's convenient to be able to change a constant in the protocol and having all the changes reflected in the overall transport.

Reply:
We can also do something like:

```rust
const _: () = assert!(BatchSize::MAX == batch_size::UNICAST);
```

Reply:
Why? That's not a requirement that the default `BatchSize` for unicast is equal to the max...

Reply:
It can be inlined next to the `manager.config.batch_size.min(link.config.batch.mtu)` so the assumption is clear.